src/Uno.UI.Runtime.Skia.Gtk/GtkCorePointerInputSource.cs

```diff
@@ -411,7 +412,7 @@ private void UseDevice(PointerPoint pointer, Gdk.Device device)
 			case EventType.Scroll when EventHelper.GetScrollDeltas(evt!, out var scrollX, out var scrollY):
 				var isHorizontal = scrollY == 0;
 				properties.IsHorizontalMouseWheel = isHorizontal;
-				properties.MouseWheelDelta = (int)(isHorizontal ? scrollX : scrollY);
+				properties.MouseWheelDelta = (int)(isHorizontal ? scrollX : -scrollY) * ScrollContentPresenter.ScrollViewerDefaultMouseWheelDelta;
```

Comment:
Is the cast intended before the multiplication? or should it be after?

Reply:
Not really, will change it. Anyway from my tests we are getting only `1` or `-1` so it should not change anything.